tgui/packages/tgui/stories/Themes.stories.tsx

```diff
@@ -6,23 +6,26 @@
 
 import { Input, LabeledList, Section } from 'tgui-core/components';
 
-import { useLocalState } from '../backend';
+import { useSharedState } from '../backend';
 
 export const meta = {
   title: 'Themes',
   render: () => <Story />,
 };
 
-const Story = (props) => {
-  const [theme, setTheme] = useLocalState('kitchenSinkTheme', undefined);
+const Story = () => {
+  const [theme, setTheme] = useSharedState<string | undefined>(
+    'kitchenSinkTheme',
+    undefined,
+  );
```

Comment:
I suspect `useState` would be fine here as well. If there's a reason it needs to be shared lmk.

Reply:
The theme story is currently broken on master and upstream. `theme` has to be set in `Themes.stories.tsx` but used in `KitchenSink.tsx`. Shared state is basically how it was able to be done pre-react.